### DISCUSSION TITLE
Is *terminal good image protocol* abandoned?

### DISCUSSION THREAD

**Discussion by timsofteng** [contour-terminal/contour#544] [Q&A] [answered]
Hello. It was cool idea but it looks like abandoned...

https://github.com/contour-terminal/terminal-good-image-protocol

**Comment by christianparpart** [comment, marked as answer]
Hey @timsofteng.

Many thanks for bringing this up, and I apologize for the severe lag of contributions and communications on this end for quite some time now. This is primarily due to my personal life's situation and the pending bigger-sized PRs I want to stabelized and finished (i.e. merged) before concentrating on the Good Image Protocol again). So this is certainly not dead from my end.

I also just released 0.2.3 of contour and the next one (0.3.0) will contain my big PR (#342) merge as I'm close to feeling confident having this one stabilized enough to be merged to master (tm). :)

**Comment by christianparpart** [comment]
@timsofteng Is it okay to convert this to a Discussion thread?

**Comment by timsofteng** [comment]
> @timsofteng Is it okay to convert this to a Discussion thread?

Sure! Totally up to you.

**Comment by timsofteng** [comment]
> Hey @timsofteng.
> 
> Many thanks for bringing this up, and I apologize for the severe lag of contributions and communications on this end for quite some time now. This is primarily due to my personal life's situation and the pending bigger-sized PRs I want to stabelized and finished (i.e. merged) before concentrating on the Good Image Protocol again). So this is certainly not dead from my end.
> 
> I also just released 0.2.3 of contour and the next one (0.3.0) will contain my big PR (#342) merge as I'm close to feeling confident having this one stabilized enough to be merged to master (tm). :)

Got it.
Hope we will see this new protocol in future!
Thanks anyway.
